### ISSUE TITLE
Usage with Mavic Mini

### ISSUE THREAD

**Issue by Jonty** [PSBPOSAS/dji-asdk-to-python#12]
I'm trying to use this with my Mavic Mini and I'm unsure if I'm either doing something daft or it's not compatible.

I have installed the DJI Android Bridge App and connected to the drone, it shows that the link is successful, but running any of the examples in this project fails as they time out as the port on my phone is closed (11111). I tried using two other ports open on my device but neither work.

Is this not compatible with DJI Fly / DJI Android Bridge App, or is there something I'm not doing correctly?

**Comment by cartovarc** [comment]
Hi! you need install dji-asdk-to-python android app on your device.  Are you using it? 

You can download it here:

https://github.com/PSBPOSAS/dji-asdk-to-python/releases/tag/v0.1.0

**Comment by Jonty** [comment]
Oh! That isn't documented in the README at all, I didn't realise - will send a PR to add it.

Where is the source code for that app, is it this? https://github.com/PSBPOSAS/aras-android-dji

**Comment by cartovarc** [comment]
No, that's an older version. For now we cannot share the source code of the android app until we have a first version on Google Play.

**Comment by JavierUR** [comment]
Hello! Do you have a new version of the app to test this?
The other link isn't available

**Comment by qinxi117** [comment]
@cartovarc Can you please post the android app's link? Thanks a lot!
